### PR TITLE
[ru] Translate monolithic-apps.md

### LINKED ISSUE #3243: [ru] Localize `Monolithic Apps`
### Description

This Issue is to localize [`Monolithic Apps`](https://github.com/cncf/glossary/blob/main/content/en/monolithic-apps.md) into Russian.

### Checklist before opening your PR (put `x` in the checkboxes) — note that you have to _add this checklist to your PR and check all the boxes for it to be approved_.
- [] This PR uses the **dev-ru** branch as a source/target.
- [] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.
    
**ETA: TBD**

Useful links:
[Как внести свой вклад](https://glossary.cncf.io/ru/contribute/)
[Руководство по стилю](https://glossary.cncf.io/ru/style-guide/)

[Slack channel](https://app.slack.com/client/T08PSQ7BQ/C05G46RMQTX)
[Telegram channel](https://t.me/ru_cloudnative)

### PR DESCRIPTION
**Pull request by Squoresha**
### Describe your changes
Localize `Monolithic Apps`

### Related issue number or link (ex: `resolves #issue-number`)
Resolves #3243 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/ru/monolithic-apps.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: Монолитные приложения
+status: Completed
+category: concept
+tags: ["architecture", "fundamental", ""]
+---
+
+Монолитное приложение состоит из компонентов, находящихся в единой кодовой базе.
```
Comment (kirkonru) [inline content/ru/monolithic-apps.md:8]:
```suggestion
Монолиты — традиционный подход к разработке программного обеспечения, при котором все компоненты приложения объединены в одну неделимую программу.
```

Comment (shurup) [reply content/ru/monolithic-apps.md]: @kirkonru the original (English) text doesn't state anything about "традиционный подход к разработке программного обеспечения". While these words are true, we have to change the original article first if we want to leave this text in our translation.

File: content/ru/monolithic-apps.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+---
+title: Монолитные приложения
+status: Completed
+category: concept
+tags: ["architecture", "fundamental", ""]
+---
+
+Монолиты — традиционный подход к разработке программного обеспечения, при котором все компоненты приложения объединены в одну неделимую программу.
+Зачастую это самый простой и понятый путь при разработке приложения.
```
Comment (shurup) [inline content/ru/monolithic-apps.md:9]: I am aware Kirill has changed "для создания" to "при разработке". However, the original text says "easiest place **to start** when making an application", and we'd better indicate that.
```suggestion
Зачастую это самый простой и понятый путь на начальном этапе разработки приложения.
```

File: content/ru/monolithic-apps.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+Однако по мере усложнения приложения монолит может стать трудным в обслуживании.
```
Comment (shurup) [inline content/ru/monolithic-apps.md:10]: "Обслуживание" is more about _running_ (or _operating_) an app, yet we're talking about _maintaining_ it.
```suggestion
Однако по мере усложнения приложения поддерживать монолит становится труднее. 
```

File: content/ru/monolithic-apps.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+Монолиты — традиционный подход к разработке программного обеспечения, при котором все компоненты приложения объединены в одну неделимую программу.
+Зачастую это самый простой и понятый путь при разработке приложения.
+Однако по мере усложнения приложения монолит может стать трудным в обслуживании. 
+С увеличением числа разработчиков, работающих над одной и той же кодовой базой, возрастает вероятность 
+конфликтующих изменений, а также увеличивается потребность в коммуникациях.
+
+## Какую проблему решает
+
+Разделение приложения на [микросервисы](/microservices-architecture/) увеличивает операционные издержки, связанные с
+тестированием, развертыванием и последующей поддержкой.
```
Comment (shurup) [inline content/ru/monolithic-apps.md:17]:
```suggestion
тестированием, развертыванием и последующей эксплуатацией.
```

**Review by shurup — COMMENTED** [review]
Great job, thank you for your first contribution! Here are just a few small suggestions from me.

File: content/ru/monolithic-apps.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+---
+title: Монолитные приложения
+status: Completed
+category: concept
+tags: ["architecture", "fundamental", ""]
+---
+
+Монолиты — традиционный подход к разработке программного обеспечения, при котором все компоненты приложения объединены в одну неделимую программу.
```
Comment (kirkonru) [inline content/ru/monolithic-apps.md:8]:
```suggestion
В монолитном приложении все его компоненты объединены в единую развертываемую программу.
```

File: content/ru/monolithic-apps.md (hunk 2 above)
Comment (kirkonru) [reply content/ru/monolithic-apps.md:9]: Thanks for the suggestion!

**Comment by kirkonru** [comment]
@shurup thanks for the great suggestions! Please approve so I can merge it.

**Review by shurup — APPROVED** [review]
Thanks, everyone! LGTM!

**Review by kirkonru — APPROVED** [review]

**Pull request merged**